### ISSUE TITLE
1.3.0: reload_mode works not correctly

### ISSUE THREAD

**Issue by neralex** [savonet/liquidsoap#441]
After the update from 1.2.1 to 1.3.0 playlist.once ignores the files in the playlist folders. The dynamic request was created but the reload ignores all files in the folders. No related errors in the log-file.

`sunday_16 = playlist.once(reload_mode="watch","/home/liquidsoap/shows/sunday_16")`

**Comment by toots** [comment]
Have you tried to install `inotify`? I can make it work this way:

```
% mkdir /tmp/playlist
% mkdir /tmp/files
% cp <mp3 files> /tmp/files
```

On another console:
```
% liquidsoap 'output.ao(fallible=true,playlist.once(reload_mode="watch","/tmp/playlist"))'
```

Then:
```
% mv /tmp/files/* /tmp/playlist
```

Result: playlist is reloaded and playback starts.

**Comment by neralex** [comment]
Ok I found the /temp/ and created the /temp/playlist and /temp/files but I'm stuck on the 2nd step with output.ao -
```
At line 1, char 10: the variable output.ao used here has not been previously
  defined.

```
BTW: I'm using two liquidsoap daemons for different stream-channels on the same box.

**Comment by toots** [comment]
You can use `output.dummy` instead of `output.ao`.

Feel free to try again `1.3.0`, it'd be nice to get to the bottom or your issue and fix these issues for further releases.

**Comment by neralex** [comment]
I would really use it in order to help you but I don't know how I should do this. The lack of documentation makes it really hard to follow. If I'm running two different daemons on my box, how should it work, when I use it as you stated? How should liquidsoap identify the correct deamon? Its a production environment with two active liquidsoap daemons. This kind of try&error is not possible with a environment like this. It is simply a logical issue for me, so I'm trying to understand what do you mean. 

In the /tmp/-folder of my debian8 installation are existing no folders like 'playlist' or 'files' and it works with 1.2.1. So the main-question is, why it works with 1.2.1 without any issues and not more with 1.3.0. Where stores liquidsoap the files instead there? Why I can't see this in the log-file?

**Comment by toots** [comment]
Well, the instructions above were just intended to show that your issue does not come from `playlist.once`, at least not the way it is reported.

Another thing that I have noticed in your script: could you try to use `fade.final` and `fade.initial` instead of `fade.out` and `fade.in` in your transitions? There were some bugs fixed in those operators in `1.3.0`..

**Comment by neralex** [comment]
Sinced I tried the rollbak to 1.2.1 reload_mode="" is completely not more working on all playlist functions - also on my 247 playlist. I tried reload=5,reload_mode="seconds" with the same result. I have to restart the deamon after each change. I could cry because it seems my whole installation sucks since the update to 1.3.0 and I don't have a clue how I can fix it.

I deinstalled 1.2.1 completely and installed 1.30 from scratch but after enter output.dummy in another shell, it it starts a shutdown bcause: " [dummy:3] Source failed (no more tracks) stopping output..." before I get a chance to move the file in the playlist folder.

I tried also to use fade.initial instead of fade.in - that works but if I use fade.final instead of fade.out liquidsoap is freezing and I need kill -9 PID to stop it. The log reported a endless loop with the output: 

`[fade_out_5751:3] Warning: #get called when not #is_ready! This is normal if an operator using this source has been unused while the source has gone unavailable. If unsure about this warning, you are very welcome to report it and ask for clarifications.`

So what can I do now? Why it ignores each reload and why also on 1.2.1 only with the difference that 1.2.1 add it all after a restart of the daemon?

**Comment by neralex** [comment]
Your suggested Inotify package was the reason why 1.2.1 wont reload. After uninstalling this package 1.2.1 was back in service as usual.

**Comment by toots** [comment]
Good! How about `1.3.0`?

**Comment by neralex** [comment]
I will test it now!

**Comment by neralex** [comment]
Reload with 1.3.0 works as expected but not the playback. The folders, which are prepared with files for the time scheduling don't show "MAD accepted" with the string of the filepath in the log-file after a restart and also not after a simple reload. So the playback wont start.

**Comment by toots** [comment]
Ok. Let's close this issue here and continue in the mailing list. Sounds good?

**Comment by toots** [comment]
Cool. Thanks for those tests.
